dev/ParallaxView/ScrollInputHelper.cpp

```diff
@@ -194,8 +194,8 @@ winrt::RichEditBox ScrollInputHelper::GetRichEditBoxParent(const winrt::Dependen
 // Returns the inner ScrollViewer or Scroller if any.
 void ScrollInputHelper::GetChildScrollerOrScrollViewer(
     const winrt::DependencyObject& rootElement,
-    _Out_ winrt::Scroller* scroller,
```

Comment:
>_Out_ [](start = 4, length = 5)

As I put in the other PR, _Out_ is a meaningful SAL annotation. I would leave these and then work with Regis when he's back to fix these up to be more modern (either ref params or return values)